_posts/2020-04-03 Hello World post.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: Hello World!
+Author: John
+---
+
+# Hello World!
+
+In case you have just joined us, CODEVID-19 was launched on March 16, 2020 in Edmonton, Canada.  It has been truly awe-inspiring to see how the hackathon has brought together developers, designers, passionate creators, and experts from public health and emergency management from around the globe. 
+
+It turns out that when you put more than 1500 intensely energized and focused people, from over 40 countries into a Slack channel, amazing things happen!  
+
+The purpose of the hackathon isn’t to win prizes, but to build useful apps to manage and survive the COVID-19 pandemic. 
```

Comment:
Dumb question, but are you sure you want to use this form of apostrophe instead of the better supported typewriter apostrophe?

Reply:
No dumb questions, otherwise I'm in serious trouble Ben.  I pasted the text from google docs. If I want to take your suggestion is it just a matter of delete and retype inside the editing window?